sycl/include/sycl/multi_ptr.hpp

```diff
@@ -1148,6 +1154,9 @@ class multi_ptr<const void, Space, access::decorated::legacy> {
 
   static constexpr access::address_space address_space = Space;
 
+  using decorated_type =
+      typename detail::DecoratedType<const void, Space>::type;
+
```

Comment:
This should be removed. It is defined publicly, but is not part of the specification.

Reply:
@steffenlarsen , thanks, done.